external-import/misp/src/misp.py

```diff
@@ -1879,9 +1879,20 @@ def prepare_elements(self, galaxies, tags, author, markings):
                     threats = self.helper.api.stix_domain_object.list(
                         types=["Intrusion-Set", "Malware", "Tool", "Attack-Pattern"],
                         filters={
-                            "mode": "and",
+                            "mode": "or",
                             "filters": [
-                                {"key": ["name", "x_mitre_id"], "values": [tag_value]}
+                                {
```

Comment:
You can write this filter in a simpler way : 
filters = {
   "mode": "and",
   "filters": [
      {"key": ["name", "x_mitre_id", "alias"], "values": [tag_value]}
   ],
   "filterGroups": [],
}

Reply:
this is equivalent to : name = tag_value OR x_mitre_id = tag_value OR alias = tag_value

Reply:
{"key": ["name", "x_mitre_id", "aliases", "x_opencti_aliases"], "values": [tag_value]}